examples/use-cases/Credit_card_fraud.ipynb

```diff
@@ -58,7 +58,7 @@
     {
      "data": {
       "text/plain": [
-       "'0.6.1'"
+       "'0.7.0'"
```

Comment:
Is it VerticaPy version? It would be nice to minimize the number of places need to be updated for each release.

Reply:
It is just an example which only works with the v0.7.0. It is not global to many files.